operations/helm/charts/grafana-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: grafana-agent
 description: 'Grafana Agent'
 type: application
-version: 0.29.0
+version: 0.30.0
```

Comment:
Let's not increment the version quite yet. There is one other pending PR for helm, and then I'll make a new release shortly.